docs/courses/level-one/chapter-5/chapter-5.6.md

```diff
@@ -0,0 +1,27 @@
+# 6. Notifying the Team
+
+In this step of the workflow you will learn how to send messages to a Discord channel using the *Discord* node.
+
+Now that you have a calculated summary of the booked orders you need to notify Nathan’s team in their Discord channel. n8n has a **Discord node** that allows you to send messages.
+
+::: tip 💡 Communication nodes
+You can replace the *Discord* node with another communication app. n8n also has nodes for [*Slack*](https://docs.n8n.io/nodes/n8n-nodes-base.slack/) and [*Mattermost*](https://docs.n8n.io/nodes/n8n-nodes-base.mattermost/).
+:::
+
+In your workflow, add a *Discord* node connected to the *Function* node. In the *Discord* node window, configure the parameters:
+::: v-pre
+- *Webhook URL:* [https://discord.com/api/webhooks/862290560479526932/qmSLXOr0RWmP9i7OSPJ-p08mFZbe4CBkxxg8IjUS0Mixxi-VjeLXONY8ZorIqjgILT0F](https://discord.com/api/webhooks/862290560479526932/qmSLXOr0RWmP9i7OSPJ-p08mFZbe4CBkxxg8IjUS0Mixxi-VjeLXONY8ZorIqjgILT0F)
+- *Text (Expression):* This week we have {{$json["bookedOrders"]}} booked orders.
```

Comment:
This week we have {{$json["bookedOrders"]}} booked orders. Secret Code: your_secret_code